examples/footloose/upload-controller-image.sh

```diff
@@ -3,4 +3,4 @@
 scriptdir="$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)"
 toolsdir=$scriptdir/../../tools
 
-$scriptdir/upload-image.sh node0 docker.io/weaveworks/wksctl-controller:$($toolsdir/image-tag) $@
+$scriptdir/upload-image.sh centos-multimaster-node0 docker.io/weaveworks/wksctl-controller:$($toolsdir/image-tag) $@
```

Comment:
Why don't we need one of these for each OS?

Reply:
we probably should. I've added it.